src/main/java/mate/academy/Main.java

```diff
@@ -2,11 +2,12 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
         // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(2); // Provide implementation that fits your needs
```

Comment:
It's a good practice to shut down the ExecutorService after its tasks have completed to free up system resources. Consider adding a shutdown hook or a try-finally block to ensure that the executor is properly shut down even if the application encounters an unexpected error.